src/commands/external.rs

```diff
@@ -1,3 +1,5 @@
+use std::{collections::HashMap, env};
```

Comment:
any reason this is separated from other imports? May be best to put inline with the others (alphabetically after `spin_plugins` imports). `cargo fmt` may do this?